Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
+FROM ubuntu:16.04
```

Comment:
why did you downgrade this?

Reply:
just for consistency with Dockerfile.gpu version and was testing different variations because of the requirements error I had before.